### DISCUSSION TITLE
After modifying a point layer on the field, I cannot package it again without every feature being deleted during packaging

### DISCUSSION THREAD

**Discussion by PandarianPwnerfluken** [opengisch/QField#4453] [Bugs]
We have a few hundred wells where we perform basic maintenance and I created a Qfield project for field reporting. The project consists of a points layer containing the wells and 7 tables for the report forms and various other purposes. During one field incursion I moved a few wells on the point layer to better reflect reality, but after that every time i tried to package the project for the next field trip it copies the point layer, but It is completely empty. I managed to figure it out that this only happens if the layer is on offline editing, but not when on copy. But if I leave it on copy It doesn't transfer the changes I made to the layer in the field. 

![image](https://github.com/opengisch/QField/assets/133377762/f0c46e60-cb99-4320-a5db-dfd50f287fad)
![image](https://github.com/opengisch/QField/assets/133377762/b4645859-f88b-4e91-9d21-ccc58f725ce7)

**Comment by nirvn** [comment]
@PandarianPwnerfluken , we'll need more details than this to be able to look into this issue.

What would help a test project / dataset with replicable steps from having a dataset with 171 points to zero.

**Reply by PandarianPwnerfluken** [reply]
Hi, Sorry for the delay, I was busy with fieldwork. 

I don't know the exact steps sadly, I was working on this project a lot to refine it, fix it. All I noticed is that after one field trip, I suddenly had this problem. The only notable thing I did was moved around a few points by a few meters.

I tried exporting the layer to another geopacked and it worked fine there, I also tried to create a new test project where i put down a few points, referenced a few tables similarly to the original and then moved them around in a packaged version, then synced it back, but that worked as well, and the issue never surfaced.
